data/json/npcs/tacoma_ranch/NPC_ranch_bartender.json

```diff
@@ -48,25 +48,21 @@
       {
         "text": "[$8] I'll take a beer",
```

Comment:
I guess we need to rename this to mention Merch and not dollars.

Reply:
He'll take $8 worth of barter.  3 Merch will do the job, but you can pay him with other stuff, so I'd prefer it stays $.